### PR TITLE
Shelf calculator/bookcase designer bugfixes

### PR DESCRIPTION
**Pull request by caseyszilagyi**
## Feature Description/Implementation
- Back button
- Rounding error
- Allowing decimals in integer fields

### CODE REVIEW

File: frontend/src/pages/casedesigner/detail/BookcaseDetail.tsx [hunk 1]
```diff
@@ -168,7 +167,13 @@ export default function BookcaseDetail() {
 
   const backButton = (
     <div className="flex col-4">
-      <BackButton className="ml-1" />
+      <Button
+        type="button"
```
Comment (nex-pixel) [inline frontend/src/pages/casedesigner/detail/BookcaseDetail.tsx:171]: This is an unnecessary change if you update the id with useEffect. This also would break from the functionality that user expects of this button that is there for other pages.

**Review by nex-pixel — CHANGES_REQUESTED** [review]

File: frontend/src/pages/casedesigner/detail/BookcaseDetail.tsx (hunk 1 above)
Comment (caseyszilagyi) [reply frontend/src/pages/casedesigner/detail/BookcaseDetail.tsx]: fixed

**Review by nex-pixel — APPROVED** [review]
